packages/grid/_modules_/grid/components/toolbar/GridFilterToolbarButton.tsx

```diff
@@ -11,9 +13,21 @@ import {
 import { gridPreferencePanelStateSelector } from '../../hooks/features/preferencesPanel/gridPreferencePanelSelector';
 import { GridPreferencePanelsValue } from '../../hooks/features/preferencesPanel/gridPreferencePanelsValue';
 import { optionsSelector } from '../../hooks/utils/optionsSelector';
+import { GridTranslationKeys } from '../../models/api/gridLocaleTextApi';
 import { GridApiContext } from '../GridApiContext';
 
+const useStyles = makeStyles(
+  (theme) => ({
+    list: {
+      margin: theme.spacing(1),
+      padding: theme.spacing(0, 1),
+    },
+  }),
+  { name: 'MuiDataGridFilterButtonTooltipList' },
```

Comment:
This should be the name of the component

```suggestion
  { name: 'MuiDataGridFilterButton' },
```